more-workshops/crud_http/get.md

```diff
@@ -2,77 +2,85 @@
 
 You've posted your first item to database. This is a great time to get the data from it!
 
-Let's add method called `retrieveListFromDataBase` in which we will perform GET method.
+Let's add method called `retrieveListFromDataBase` in which we will perform **GET** method.
 
-```javascript
+```typescript
     this.http.get<TodoItem[]>('http://localhost:3000/items').subscribe();
 ```
 
 As you've probably noticed `url` part is also needed here and we subscribed response of this call. Also we added `<TodoItem[]>` part which is telling that our response will have type `TodoItem[]`.
 
 But before we move on, we need to make changes to the model. In the DB, each item has a unique id \(string\) and we need to add it into our model.
 
-In `todo-item.ts` simply add `_id: string;`:
+In `todo-item.ts`, add `_id: string;`:
 
-```javascript
+```typescript
 export interface TodoItem {
     _id?: string;
     title: string;
     completed?: boolean;
 }
 ```
 
-Now the most tricky part: how can we see the data in our rendered app?
+How can we see the data in our rendered app? 🤔
 
-We will setup the whole integration by using `Subject` and `Observable` - do not worry, later on we will explain it more.
+Let's start with seeing the data we receive in DevTools console. To do so, add `console.log()` inside the `subscribe`:
 
-For starters you may want to see in DevTools console what we actually receive. To do this inside subscribe you may add `console.log()`:
-
-```javascript
+```typescript
   retrieveListFromDataBase() {
     this.http.get<TodoItem[]>('http://localhost:3000/items').subscribe(
       response => console.log(response)
     );
   }
 ```
 
-Now comes the time for a small explanation.
+How can we add the data to the application view?
+
+## Interacting with server data 
+
+We will need to use libraries that allow us to work with data when that data isn't immediately available to the application. The response from the server is asynchronous, meaning some time might pass between the request and response, but the application code isn't waiting for it. Imagine our application is multi-tasking just like we do every day! We can't always wait for something to complete before starting the next task. For example, we'll start making coffee and while the coffee is brewing, we'll get our favorite coffee cup out and maybe even start preparing breakfast before the coffee is ready to drink. 
+
+In this same way, we can write code that allows us to react to data as it comes back from the server. We'll use specialized libraries called `Subject` and `Observable`.
```

Comment:
I would suggest to introduce RxJS in here as it is a library, so we may write sth like 
> We'll use specialized library called  `RxJS` from which in Angular we are able to borrow special types `Subject` and `Observable`.

Reply:
@pelagia123 update made. Would you mind taking a look? TY!